gprimitives/src/lib.rs

```diff
@@ -119,6 +119,14 @@ impl From<H160> for ActorId {
     }
 }
 
+impl From<ActorId> for H160 {
```

Comment:
better to impl method on ActorId or impl TryInto, since its only applicable in eth scenario, where first 12 bytes are meaningless and zeroed